float64 position_gain
float64 orientation_gain
